contracts/src/libraries/xChain/T1XChainReader.sol

```diff
@@ -83,14 +113,28 @@ contract T1XChainReader is OwnableUpgradeable, ReentrancyGuardUpgradeable {
         prover = _prover;
     }
 
+    /**
+     * @notice Initializes the contract
+     * @param _owner The owner of the contract
+     */
+    function initialize(address _owner) external initializer {
+        __Ownable_init();
```

Comment:
Seems like this files is in an older state than canary. We need to merge canary in, there were a lot of changes since then.